voltron/plugins/view/register.py

```diff
@@ -367,10 +536,7 @@ def render(self):
                         if fmt['value_format'] != None and isinstance(formatted_reg, Number):
                             formatted_reg = fmt['value_format'].format(formatted_reg)
                         if fmt['value_func'] != None:
-                            if isinstance(fmt['value_func'], STRTYPES):
-                                formatted_reg = eval(fmt['value_func'])(formatted_reg)
-                            else:
-                                formatted_reg = fmt['value_func'](formatted_reg)
+                            formatted_reg = fmt['value_func'](self, formatted_reg)
                         if fmt['value_colour_en']:
```

Comment:
No. The reason for this is that the register format config sections may have any combination of those fields.